src/EditorFeatures/Test/CodeFixes/ErrorCases/CodeFixExceptionInFixableDiagnosticIds2.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.CodeFixes.ErrorCases
+{
+    public class ExceptionInFixableDiagnosticIds2 : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds => new ImmutableArray<string>();
```

Comment:
This was written because I would expect people to accidentally call `new  ImmutableArray<string>()` expecting it to produce a valid `ImmutableArray<string>`.  In terms of the test why is `default(ImmutableArray<string>)` going to be superior?